.github/dependabot.yml

```diff
@@ -4,13 +4,36 @@ updates:
   - package-ecosystem: cargo
     directory: /
     schedule:
-      interval: weekly
-      day: sunday
+      interval: daily
```

Comment:
The idea was to merge one PR per week, so how about leaving the weekly interval.

Reply:
Here's the reasoning for daily interval: previously, we had weekly interval and max 5 concurrent PRs raised (due to default depandabot settings we did not touch). Now we'll have 1 PR with all deps but 5 times, so computation-wise for CI it is the same. The problem I see with weekly interval is, at least in my experience, such PRs get outdated - either this dependency is fixed by a different team or `main` is way ahead and are conflicts etc. Having daily dependabot PR would help here given we can merge such PR in one day, so this can be a disadvantage as well.

Reply:
Personally, I used the strategy to merge on Monday and don't care for the rest of the week, but maybe I'm too lazy. :stuck_out_tongue:  
Feel free to resolve if you disagree.

Reply:
Let's try this approach and if does not work for the team, we'll revert to Weekly.